### DISCUSSION TITLE
Server error

### DISCUSSION THREAD

**Discussion by VFMick** [netbox-community/netbox#10331] [General]
After a new install i try to connect to the web interface end get the following error: -

Server Error

There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'redis.exceptions.ConnectionError'>

Error 111 connecting to localhost:6379. Connection refused.

Python version: 3.10.4
NetBox version: 3.3.2

does anybody have an idea what the issue might be?

**Comment by candlerb** [comment]
Netbox depends on redis.  Perhaps you didn't install it, or it's not running.  See https://docs.netbox.dev/en/stable/installation/2-redis/

Usual debugging: `systemctl status redis-server`, `journalctl -eu redis-server` etc.

**Reply by VFMick** [reply]
you were spot on, installed redis and it works now, the instructions i followed missed that out. 
Thanks for the quick response :-)
